src/Parser/Core/Modules/SpellUsable.test.js

```diff
@@ -360,6 +360,15 @@ describe('Core/Modules/SpellUsable', () => {
         });
       }
     });
+    it('A charge spell that has the cooldown of more than a single charge reduced should return the full reduction value', () => {
```

Comment:
I think a better position for this test would be underneath the `reducing a cooldown beyond its duration finishes the cooldown` test.

Reply:
For name I recommend to state how it should work, preferably like it works in-game. e.g. `reducing a spell with multiple charges on cooldown reduces the CD time on the next charge if it fully recharges the first charge`. The full reduction value should always be returned by this method and is tested separately.